contentcuration/contentcuration/frontend/shared/views/TipTapEditor/TipTapEditor/TipTapEditor.vue

```diff
@@ -156,7 +222,7 @@
 
   .editor-container {
     position: relative;
-    width: 1000px;
```

Comment:
Removing the width here made the container not grow to fit the dev-harness so the editor doesn't grow beyond 200px on my screen.

<img width="1081" height="474" alt="image" src="https://github.com/user-attachments/assets/86d9f1ea-7b4c-45fc-83b6-269b32550eb4" />